README.md

```diff
@@ -0,0 +1,33 @@
+## START SERVER
+Use ```foreman start```
```

Comment:
- [ ] Your app has no tests.  Please, kindly add some tests in order to ensure your app's integrity 💪